.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   base:
     docker:
-      - image: circleci/node:14
+      - image: cimg/node:14.18
```

Comment:
Could we set this to the same value specified in https://github.com/PagerDuty/pdjs/blob/main/.tool-versions ? Maybe update them both to 16.13.2?